_rules/device-motion-disabled-c249d5.md

```diff
@@ -0,0 +1,226 @@
+---
+id: c249d5
+name: Device motion based changes to the content can be disabled
+rule_type: atomic
+description: |
+  This rule checks that it is possible to disable any changes to the content of the web page resulting from device motion based events.
+accessibility_requirements:
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'attr-lowercase'
+---
+
+## Applicability
+
+The rule applies to an [HTML document][] with an associated [Window object][] that has an [event listener list][] with one or more [event listeners][event listener] for [device orientation events][device orientation] or [device motion events][device motion].
+
+## Expectation
+
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target one of the following is true:
+
+- **no changes:** The registered event does not cause [changes to the content][changes in content] of the [web page][] within a 10 minute time span of the [event firing][event firing]; or
+- **disabled:** There is at least one set of [instruments][instrument], where each [instrument][] is in the same [web page][] of the registered event or can be found in a [clearly labeled location][] from that [web page][], to prevent [changes to the content][changes in content] of the [web page][] resulting from the event within a 10 minute time span of the [event firing][].
```

Comment:
Minor editorial suggestion:

```suggestion
- **disabled:** There is at least one set of [instruments][instrument], where each [instrument][] is in the same [web page][] of the registered event or can be found in a [clearly labeled location][] from that [web page][]. The set of [instruments][] can be used to prevent [changes to the content][changes in content] of the [web page][] resulting from the event within a 10 minute time span of the [event firing][].
```